package.json

```diff
@@ -100,7 +100,7 @@
     "bump": "npm version -m 'build: %s release'",
     "debug": "ts-node src/cli.ts",
     "lint": "eslint . bin/rdme --ext .js,.ts",
-    "lint-docs": "npx alex . && npm run prettier:docs",
+    "lint-docs": "npx -y alex . && npm run prettier:docs",
```

Comment:
Changed this because I hate running `npm test`, clicking into a new window somewhere and then realizing that it's sitting there asking me to install alex instead of running tests.

Reply:
nevermind i guess this doesn't work on node 14. blimey